packages/paste-core/components/detail-text/stories/index.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import type {StoryFn} from '@storybook/react';
+import {Box} from '@twilio-paste/box';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {useTheme} from '@twilio-paste/theme';
+
+import {DetailText} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/Detail Text',
+  component: DetailText,
+};
+
+export const Default: StoryFn = () => (
```

Comment:
question: could there any benefit to having stories with Details Text `as=somethingelse` ? I guess not since these are just for visual testing?

Reply:
Yeah, I just left that for the unit tests. I'm open to adding a story tho if itll add value